### PR TITLE
Pull request for Issue#42, Team 1

### PR DESCRIPTION
**Pull request by NejcZupec**
Now when user login, he is redirected back from where he came.

### CODE REVIEW

File: piplmesh/account/templates/login.html [hunk 1]
```diff
@@ -12,6 +12,7 @@
                 {% csrf_token %}
                 {{ form.as_p }}
                 <p><button type="submit" class="auth_button" value="value">{% trans "Login" %}</button></p>
+                <input type="hidden" name="next" value="{{ next }}" />
```
Comment (mitar) [inline piplmesh/account/templates/login.html:15]: OK. You are right. login view populates this variable.

Comment (NejcZupec) [inline piplmesh/account/templates/login.html:15]: :)

File: piplmesh/frontend/templates/header.html [hunk 2]
```diff
@@ -36,7 +36,7 @@
                     </form>
                 </li>
             {% else %}
-                <li><a href="{% url login %}">{% trans "Log in" %}</a></li>
+                <li><a href="{% url login %}?next={{ request.get_full_path }}">{% trans "Log in" %}</a></li>
```
Comment (mitar) [inline piplmesh/frontend/templates/header.html:39]: But you cannot hardcode `next` - you should use the value from `django.contrib.auth.REDIRECT_FIELD_NAME`.

File: piplmesh/settings.py [hunk 3]
```diff
@@ -176,7 +176,7 @@
     }
 }
 
-LOGIN_REDIRECT_URL = '/'
+#LOGIN_REDIRECT_URL = '/'
```
Comment (mitar) [inline piplmesh/settings.py]: Remove it or leave it. Not just comment it out (especially not without an explanation).

Probably there is nothing wrong with this left as it is, it is then a fallback URL to redirect.

File: piplmesh/frontend/templates/header.html (hunk 2 above)
Comment (mitar) [inline piplmesh/frontend/templates/header.html:39]: TODO.

File: piplmesh/account/templates/login.html (hunk 1 above)
Comment (mitar) [inline piplmesh/account/templates/login.html:15]: It is not XHTML valid. Also check in current main repository how to get better `next` value (called `next_url`).

**Comment by mitar** [comment]
Update pull request to latest from main repository, please.

File: piplmesh/settings.py (hunk 3 above)
Comment (NejcZupec) [inline piplmesh/settings.py]: Ok, I will leave it.

**Comment by mitar** [comment]
Reopen. :-)

**Comment by mitar** [comment]
And finish your work!

**Pull request closed without merging**